### DISCUSSION TITLE
Help with Running Taiko in Chrome?

### DISCUSSION THREAD

**Discussion by jaydee713** [getgauge/taiko#2061] [Development]
Hi! I'm trying to run `npx taiko` on MacOS (intel, if it matters) with a Chrome browser, but every time the browser opens, taiko can't seem to see the target: 

```
$ TAIKO_BROWSER_PATH='/Applications/Google Chrome.app/Contents/MacOS/Google Chrome' npx taiko

Version: 1.2.5 (/Applications/Google Chrome.app/Contents/MacOS/Google Chrome)
Type .api for help and .exit to quit

> openBrowser()
 ✘ Error: No targets created yet!, run `.trace` for more info.
```
Yet even though I don't have a target, I can't open another browser with `openBrowser()`, _and_ I can't `closeBrowser()`, and finally `.exit` doesn't complete:

```
> openBrowser()
 ✘ Error: openBrowser cannot be called again as there is a browser instance open., run `.trace` for more info.
> closeBrowser()
WARNING: Browser or page not initialized. Call `openBrowser()` before using this API
> .exit



^C
```

(Also of note, I also have a dialog that pops up asking me to send diagnostics info + set default browser...)

Anybody know how to fix this so that I can swap out the default Chromium to Chrome?

TIA!

**Comment by jaydee713** [comment]
For the next person... after scouring the internet, I wasn't able to fix this and swapped to using Selenium 4.0 instead.

**Comment by DevaIrakkamRubavathy** [comment]
Hi @jaydee713 ,
`TAIKO_BROWSER_PATH=/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome  TAIKO_BROWSER_ARGS='--no-first-run' npx taiko`
I tried on MacOS (Intel core i7) with Google Chrome Version 91.0.4472.114 (x86_64), node version 12.22.1 and npm version 6.14.12
Try this command. It might help you to run taiko in chrome browser.
